### PR TITLE
Fix playback status in android app when reconnecting

### PR DESCRIPTION
**Pull request by Froghut**
When the android app was restarted while playback was paused on chromecast it would not show that something was playing on chromecast in the app.

### CODE REVIEW

File: components/maincontroller.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿define(['datetime', 'embyactions', 'browserdeviceprofile', '//www.gstatic.com/cast/sdk/libs/receiver/2.0.0/cast_receiver.js', '//www.gstatic.com/cast/sdk/libs/mediaplayer/1.0.0/media_player.js'], function (datetime, embyActions, deviceProfileBuilder) {
+define(['datetime', 'embyactions', 'browserdeviceprofile', '//www.gstatic.com/cast/sdk/libs/receiver/2.0.0/cast_receiver.js', '//www.gstatic.com/cast/sdk/libs/mediaplayer/1.0.0/media_player.js'], function (datetime, embyActions, deviceProfileBuilder) {
```
Comment (Froghut) [inline components/maincontroller.js]: No idea why this line shows as changed, I did not touch it and can't see the difference

File: components/maincontroller.js [hunk 2]
```diff
@@ -149,6 +149,10 @@
 
         clearMediaElement();
 
+        window.playlist = [];
+        window.currentPlaylistIndex = -1;
+        embyActions.displayUserInfo($scope, $scope.serverAddress, $scope.accessToken, $scope.userId);
+
```
Comment (Froghut) [inline components/maincontroller.js]: when stopping playback reset playlist etc. so isPlaying() is not returning true

File: components/maincontroller.js [hunk 3]
```diff
@@ -273,6 +277,9 @@
             if (!isPlaying()) {
                 embyActions.displayUserInfo($scope, data.serverAddress, data.accessToken, data.userId);
             }
+            else {
+                embyActions.reportPlaybackProgress($scope, getReportingParams($scope), true, reportEventType);
+            }
```
Comment (Froghut) [inline components/maincontroller.js]: when a client reconnects it sends the 'Identify' command, and if something is currently playing (or paused) report playbcack progress back so the UI in the app is updated correctly and shows the currently playing item as playing.

File: components/maincontroller.js (hunk 1 above)
Comment (dkanada) [reply components/maincontroller.js]: probably just a newline change, can be fixed with an editor config

**Comment by anthonylavado** [comment]
@Froghut Once we get that line change fixed up, we'll push it through the rest of the process.

**Comment by Froghut** [comment]
I changed the file encoding to UTF8 BOM, that removed the changed first line.

**Comment by anthonylavado** [comment]
@froghut oops, there's a conflict now, sorry :-(

**Comment by Froghut** [comment]
This is actually from an earlier PR from me, I think the best thing would be a kind of merge of the two:
`else {
                embyActions.reportPlaybackProgress($scope, getReportingParams($scope), true, "playbackstop");
            }`
Is this something I can/should do?

**Pull request merged**